config/secrets.yaml.template

```diff
@@ -11,6 +11,8 @@ storage:
     storageaccountsuffix: core.windows.net
 
 default:
-    # SSH public key used to create a user and connect to a server.
-    # It can either be the public key itself(ssh-rsa ...) or the path to the ssh key.
+    # SSH keys used to create a user and connect to a server.
+    # The public key can either be the public key itself(ssh-rsa ...) or the path to the ssh key.
+    # The private key must be the path to the key.
```

Comment:
do we require them to enter a private ssh key even if its called *id_pub*?

Reply:
no, ssh_priv_key is entirely optional. If it is not specified, the ssh client they are using will still look for a private key using the standard paths & file names it expects (which includes id_pub in the ~/.ssh directory.

Reply:
We should document that it is optional and perhaps even remove it from the template and only have that in documentation.

Reply:
yeah -- I think it is a real edge case that it would be used.